server/server.go

```diff
@@ -1400,3 +1400,7 @@ func (s *Server) SaveTTLConfig(data map[string]interface{}, ttl time.Duration) e
 	}
 	return nil
 }
+
+func (s *Server) SplitAndScatterRegions(context context.Context, r *pdpb.SplitAndScatterRegionsRequest) (*pdpb.SplitAndScatterRegionsResponse, error) {
```

Comment:
@rleungx  @LemonHX  who/which PR update the `kvproto `but doesn't update in PD?

Reply:
https://github.com/tikv/pd/pull/4389 has not been merged yet.